google/cloud/aiplatform/models.py

```diff
@@ -50,18 +57,22 @@ def __init__(
         location: Optional[str] = None,
         credentials: Optional[auth_credentials.Credentials] = None,
     ):
-        """Retrieves the model resource and instanties it's representation.
+        """Retrieves the model resource and instantiates it's representation.
```

Comment:
nit: The apostrophe is unnecessary. 'Retrieves the model resource and instantiates its representation'